website/content/commands/kv/get.mdx

```diff
@@ -7,11 +7,21 @@ page_title: 'Commands: KV Get'
 
 Command: `consul kv get`
 
+Corresponding HTTP API Endpoint: [\[GET\] /v1/kv/:key](https://www.consul.io/api-docs/kv#read-key)
```

Comment:
We should include the ACL Required info on the `consul kv export` and `consul kv import` pages. They don't really have a true corresponding API endpoint (though they use the get/put endpoints under the hood), but they do have ACLs required.

Reply:
👍 
I added the same ACL info for import as put and export as get.